build.rs

```diff
@@ -105,11 +142,34 @@ impl AccountType {
 		format!("{}Account", inflector::cases::pascalcase::to_pascal_case(&self.network))
 	}
 
+	fn tokens(&self, all: &[TokenType]) -> Vec<Ident> {
```

Comment:
Does that makes that much sense to pass there all tokens? Isn't this basically just:
```
self.symbols.iter().map(|s| format_ident!("{}", inflector::cases::pascalcase::to_pascal_case(s))).collect()
```

Reply:
Yeah, I just tried to stay DRY in a not performance-sensitive part of the code-base. Two different parts of the code would need to be changed in sync when we have to touch the "symbol" -> "enum variant identifier" relation. I will extract a static method for that and then we have advantages from both worlds.